src/components/game/game.tsx

```diff
@@ -0,0 +1,72 @@
+import { GameFieldComponent } from "@components/game-field/game-field";
+import { GameOverComponent } from "@components/game-over/game-over";
+import { StatisticComponent } from "@components/statistic/statistic";
+import { GameState } from "@interfaces/index";
+import { effect, untracked } from "@preact/signals";
+import { GameStateService } from "@services/game-state.service";
+import { StatisticService } from "@services/statistic.service";
+import { ThemeService } from "@services/theme.service";
+import { isRunningInTelegram } from "@utils/telegram.utils";
+import {
+  GameStateContext,
+  StatisticContext,
+  ThemeContext,
+} from "../../interfaces/context";
+import "./game.css";
+
+type GameStateComponent = typeof GameFieldComponent | typeof GameOverComponent;
+
+const statesComponents = new Map<GameState, GameStateComponent>([
```

Comment:
Using Map struct to keep links to components is not a standard practice in React.